src/core/PerspectivismCore.ts

```diff
@@ -140,8 +134,31 @@ export default class PerspectivismCore {
         return sharedPerspective
     }
 
-    createUniqueHolochainExpressionLanguageFromTemplate(languagePath: string, dnaNick: string, encrypt: Boolean, passphrase: string): Promise<LanguageRef> {
-        return this.#languageFactory.createUniqueHolochainExpressionLanguageFromTemplate(languagePath, dnaNick, encrypt, passphrase)
+    async installSharedPerspective(sharedPerspectiveUrl: string, sharedPerspective: SharedPerspective): Promise<PerspectiveID> {
```

Comment:
Those two parameters are redundant. This function should only receive the URL and then get the sharedPerspective expression from the perspective language.